documentation/IDTA-01002-3/modules/ROOT/pages/http-rest-api/http-rest-api.adoc

```diff
@@ -175,7 +175,10 @@ Technically, the idShort path is a string and the idShorts are separated by a do
 
 .Example Hierarchy of Submodel Elements
 [[se-hierarchy]]
-image::se-hierarchy.jpeg[image24,width=642,height=367]
+[plantuml, se-hierarchy, svg]
```

Comment:
This figure is not really and improvement. Since it is UML it needs to be correct. We did not introduce UML instance diagrams (in our Annex) so far in out specifications. So we should use the original figure, not a puml figure because its intentation was to just illustrate how it is done.

Of course there is a bug in the figure since it still talks about "SubmodelElementStruct" that is "SubmodelElementCollection" now.

Reply:
The original image is used now: https://github.com/admin-shell-io/aas-specs-api/pull/310/commits/677405d6cf85fece77230a1b9a0cd4c04bad3277